<h2>Description and display conventions</h2>

<p>Nuclear mitochondrial DNA segments(NUMTs) are a kind of insertion from the mitochondrion to the nucleus, which is an ongoing and frequent process that happens in all eukaryotes. In previous studies, NUMTs have been reported to increase genetic diversity, promote gene and genome evolution, and generate novel nuclear exons. NUMTs can also affect the accuracy when nuclear genomes are assembled.</p>
<p>This track is a collection of Nuclear mitochondrial DNA segments, provided in BED format.</p>
<b>Notice: Alignments to incompletely assembled or unmapped chromosome locations are omitted in this track.</b>
<p>In this track, the BED score is calculated by -10log10(E-value), representing the alignment confidence and is reflected in the level of gray, and scores >=100 (E-values <= 1e-10) are black. It is important to note that when a NUMT is a merged result, the score is taken as the highest score among all results.</p>

<h2>Comparison with 2011 human UCSC dataset</h2>
Our track offers NUMTs both in hg19 and hg38 assemblies. The exsiting human UCSC database is based on hg19 assembly. We conducted a comparison between our track and the exsiting human UCSC database using bedtools v2.31.1. The results are shown in the following table:


<table style="border: 2px solid black;">
  <thead>
      <tr>
          <th>Track</th>
          <th>Total Counts</th>
          <th>Number of 1bp overlap</th>
          <th>Number of 50% overlap</th>
          <th>Number of 90% overlap</th>
      </tr>
  </thead>
  <tbody>
      <tr>
          <th scope="row">2011 hg19 NUMTs track</th>
          <td>766</td>
          <td>-</td>
          <td>-</td>
          <td>-</td>
      </tr>
      <tr>
        <th scope="row">This hg19 NUMTs track</th>
        <td>1077</td>
        <td>756</td>
        <td>753</td>
        <td>740</td>
      </tr>
  </tbody>
</table>

<h2>Methods</h2>

<p>For more detailed information on the methods used for detecting NUMTs, please visit the following webpage:</p>
<a href="https://github.com/Koumokuyou/NUMTs">https://github.com/Koumokuyou/NUMTs</a>

<h2>References</h2>

<ol>
  <li>Kleine T, Maier UG, Leister D. DNA transfer from organelles to the nucleus: the idiosyncratic genetics of endosymbiosis. Annu Rev Plant Biol. 2009;60:115-138. doi:10.1146/annurev.arplant.043008.092119</li>
  <li>Hamada M, Ono Y, Asai K, Frith MC. Training alignment parameters for arbitrary sequencers with LAST-TRAIN. Bioinformatics. 2017;33(6):926-928. doi:10.1093/bioinformatics/btw742</li>
  <li>Zhang GJ, Dong R, Lan LN, Li SF, Gao WJ, Niu HX. Nuclear Integrants of Organellar DNA Contribute to Genome Structure and Evolution in Plants. Int J Mol Sci. 2020;21(3):707. Published 2020 Jan 21. doi:10.3390/ijms21030707</li>
  <li>Yao Y, Frith MC. Improved DNA-Versus-Protein Homology Search for Protein Fossils. IEEE/ACM Trans Comput Biol Bioinform. 2023;20(3):1691-1699. doi:10.1109/TCBB.2022.3177855</li>
</ol>
